src/services/api/authUser.js

```diff
@@ -16,4 +16,12 @@ export default {
   delete (id) {
     return axios.delete(`/api/auth/user/`)
   },
+
+  verifyMail (key) {
+    return axios.post('/api/auth/user/verify_mail/', { key })
+  },
+
+  resendVerificationRequest () {
+    return axios.post('/api/auth/user/resend_verification/')
+  },
```

Comment:
I would make these use async/await, without returning anything, otherwise some API methods return the data, yet these ones the full response object.

The use of them does not use the return data, so it should not provide it.

Reply:
I agree, but I didn't want to make any unrelated changes except for moving to the new API. I'll write it onto my list of things to do.